### PR TITLE
[algoliasearch] disableTypoToleranceOnAttributes to string array

### PR DESCRIPTION
**Pull request by tijmenbruggeman**
Please fill in this template.

- [/] Use a meaningful title for the pull request. Include the name of the package modified.
- [/] Test the change in your own code. (Compile and run.)
- [/] Add or edit tests to reflect the change. (Run with `npm test`.)
- [/] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [/] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [/] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [/] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://www.algolia.com/doc/api-reference/api-parameters/disableTypoToleranceOnAttributes/>>
- [/] Increase the version number in the header if appropriate.

### CODE REVIEW

File: types/algoliasearch/index.d.ts [hunk 1]
```diff
@@ -1655,7 +1655,7 @@ declare namespace algoliasearch {
      * default: ""
      * https://github.com/algolia/algoliasearch-client-js#disabletypotoleranceonattributes
      */
-    disableTypoToleranceOnAttributes?: string;
+    disableTypoToleranceOnAttributes?: string[];
```
Comment (keichinger) [inline types/algoliasearch/index.d.ts]: Hi @tijmenbruggeman, thank you for your contribution!

Could you please update the link to the docs (in the comment above) and also apply these changes in the `lite`-version?

**Review by keichinger — CHANGES_REQUESTED** [review]

File: types/algoliasearch/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for algoliasearch-client-js 3.30.1
+// Type definitions for algoliasearch-client-js 3.30.8
```
Comment (Haroenv) [inline types/algoliasearch/index.d.ts]: this change isn't necessary, it's only "earliest version supporting these typings"

File: types/algoliasearch/index.d.ts (hunk 1 above)
Comment (tijmenbruggeman) [reply types/algoliasearch/index.d.ts]: Hey @keichinger,

I have updated the jsdoc above. The lite-version already had the correct type definition.

**Comment by Haroenv** [comment]
Sorry @tijmenbruggeman that we forgot about your changes here. It looks however that these changes have since been made in another PR: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/algoliasearch/index.d.ts#L1146

so this PR can be closed.

**Pull request closed without merging**